sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/Identity.java

```diff
@@ -169,6 +177,7 @@ protected synchronized void applyEnabledState(boolean enabled) {
 
             /* Download the latest configuration in background. */
             downloadConfiguration();
+            mTokenStorage.cacheToken();
```

Comment:
This is not downloading the latest config, need a new comment before. Also should be done before downloading.